### PR TITLE
[IMP] stock_available_to_promise_release: Extract method to check pickings in progress

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: stock_available_to_promise_release/models/stock_move.py [hunk 1]
```diff
@@ -85,6 +87,25 @@ def _is_unreleaseable(self):
...
+        - and if the move's picking is not printed
+        - and the move has no quantity done.
+
+        """
+        moves = self.filtered(lambda m: m.state not in ("done", "cancel"))
+        if not moves:
+            return False
+        picking_ids = moves.mapped("picking_id")
+        if picking_ids.filtered("printed"):
+            return True
```
Comment (rousseldenis) [inline stock_available_to_promise_release/models/stock_move.py]: @jbaudoux I'm wondering if I should return the recordset in order to be able to get a comprehensive message to user (with picking names)

Comment (rousseldenis) [reply stock_available_to_promise_release/models/stock_move.py]: Done

File: stock_available_to_promise_release/models/stock_move.py [hunk 2]
```diff
@@ -61,8 +63,10 @@ def _compute_unrelease_allowed(self):
                 iterator = move._get_chained_moves_iterator("move_orig_ids")
                 next(iterator)  # skip the current move
                 for origin_moves in iterator:
-                    unrelease_allowed = move._is_unrelease_allowed_on_origin_moves(
-                        origin_moves
+                    unrelease_allowed = not origin_moves._in_progress_for_unrelease()
+                    unrelease_allowed = (
+                        unrelease_allowed
+                        and move._is_unrelease_allowed_on_origin_moves(origin_moves)
```
Comment (jbaudoux) [inline stock_available_to_promise_release/models/stock_move.py:69]:
```suggestion
                    unrelease_allowed = (
                        not origin_moves._in_progress_for_unrelease()
                        and move._is_unrelease_allowed_on_origin_moves(origin_moves)
```

**Review by jbaudoux — APPROVED** [review]

**Review by lmignon — APPROVED** [review]
Same comment as @jbaudoux

**Pull request merged**